frontend/views/shared/_pagination_summary.html.erb

```diff
@@ -0,0 +1,98 @@
+<% if @search_data %>
+  <div class="pull-left">
+    <%= I18n.t "pagination.summary_prefix" %> <strong><%= @search_data["offset_first"] %></strong> <%= I18n.t "pagination.summary_offset_connector" %> <strong><%= @search_data["offset_last"] %></strong> <%= I18n.t "pagination.summary_total_connector" %> <strong><%= @search_data["total_hits"] %></strong> <%= I18n.t "pagination.summary_suffix" %>
+
+    <%# PLUGIN CONTENT STARTS HERE %>
+    <% if params['controller'] == 'accessions' || Array(params['filter_term']).any? { |t| t.match('accession') } %>
+      &nbsp;
+      <span class="btn-group">
+        <%= link_to I18n.t("download_accessions.button_label"),
+          {
+           :controller => :download_accessions,
+           :action => :download,
+           :filter_term => @search_data[:criteria]['filter_term[]'],
+           :q => @search_data[:criteria]['q'],
+           :aq => @search_data[:criteria]['aq']
+          },
```

Comment:
In conjunction with search params changes above, `build_search_params.merge({:controller => :download_accessions, :action => :download})` would build a URL with all the available search params.